packages/legend-application-studio/src/components/editor/edit-panel/service-editor/ServiceEditor.tsx

```diff
@@ -445,6 +454,16 @@ export const ServiceEditor = observer(() => {
                   })}
                 >
                   {prettyCONSTName(tab)}
+                  {tab === SERVICE_TAB.TEST && (
+                    <button
+                      className="service-editor__tab__hint"
+                      tabIndex={-1}
+                      onClick={seeDocumentation}
+                      title="Click to see more details on advanced search"
```

Comment:
Maybe like click to see more details on service test?

Reply:
hhahah yes copy and paste